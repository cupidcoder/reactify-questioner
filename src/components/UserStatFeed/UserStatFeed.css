.UserStatFeed {
  padding: 1rem;
}

.UserStatFeed h4 {
  font-size: 1.5rem;
  font-style: italic;
}

.UserStatFeed h3 {
  font-size: 1.2rem;
}

