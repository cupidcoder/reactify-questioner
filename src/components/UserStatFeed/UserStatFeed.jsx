import React from 'react';
import FeedList from '../FeedList/FeedList';
import style from './UserStatFeed.css';

const UserStatFeed = () => (
  <div className={style.UserStatFeed}>
    <h4>Questions feed</h4>
    <FeedList />
  </div>
);

export default UserStatFeed;
