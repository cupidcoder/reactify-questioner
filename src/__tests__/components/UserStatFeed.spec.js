import React from 'react';
import { shallow } from 'enzyme';
import UserStatFeed from '../../components/UserStatFeed/UserStatFeed';

describe('UserStatFeed', () => {
  it('Should correctly save snapshot of the UserStatFeed', () => {
    const wrapper = shallow(<UserStatFeed />);
    expect(wrapper).toMatchSnapshot();
  });
});
