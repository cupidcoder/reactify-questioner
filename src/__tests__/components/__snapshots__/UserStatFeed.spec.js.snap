// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserStatFeed Should correctly save snapshot of the UserStatFeed 1`] = `ShallowWrapper {}`;
